cmd/server-mux.go

```diff
@@ -441,6 +442,8 @@ func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
 				RawQuery: r.URL.RawQuery,
 				Fragment: r.URL.Fragment,
 			}
+			lmsg := fmt.Sprintf("redirect for %s", u.String())
```

Comment:
Changing log level is fine but we are not going to additional logs in this manner.  Logging requires careful changes it can easily become unmanageable without proper structure.  

We are planning to fix logging in a proper manner by bringing a more well defined framework where each log message is consistent and documented.